modules/tinymce/src/core/main/ts/fmt/CaretFormat.ts

```diff
@@ -217,7 +217,7 @@ const applyCaretFormat = (editor: Editor, name: string, vars?: FormatVars): void
     selectionRng.collapse(true);
 
     // Expand the range to the closest word and split it at those points
-    let rng = ExpandRange.expandRng(editor, selectionRng, formatList);
+    let rng = ExpandRange.expandRng(editor.dom, selectionRng, formatList);
```

Comment:
```suggestion
    let rng = ExpandRange.expandRng(dom, selectionRng, formatList);
```
Yet another one I missed sorry @HAFRMO 😞 this should be it but I'd suggest double checking.

Reply:
This one doesn't seem to be right? There's no previously defined `dom` variable in this function?

Reply:
Ahh sorry I was looking at the GitHub UI and it was showing the following and I missed the line numbers:
```
const insertCaretContainerNode = (editor: Editor, caretContainer: Node, formatNode: Node) => {
  const dom = editor.dom;
```
You're correct either way so ignore this.